fuzzers/forkserver_libafl_cc/src/main.rs

```diff
@@ -145,7 +146,12 @@ pub fn main() {
     .unwrap();
 
     // The Monitor trait define how the fuzzer stats are reported to the user
-    let monitor = SimpleMonitor::new(|s| println!("{s}"));
+    let monitor = SimpleMonitor::with_user_monitor(
+        |s| {
+            println!("{s}");
+        },
+        true,
```

Comment:
Open question, does this bool really make sense in the API here? Why would you ever use this monitor with `false`?

Reply:
yeah we should this boolean